otpcasestudy-angular-app/src/app/components/admin/admin-navbar/admin-navbar.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-toolbar class="admin-navbar">
+  <span class="navbar-title">{{ title }}</span>
+  <span class="spacer"></span>
+  <button mat-flat-button *ngIf="!isOnDashboard()" (click)="goToDashboard()">
```

Comment:
```suggestion
  <button mat-flat-button *ngIf="!isOnDashboard()" routerLink="/admin/dashboard">
```

and remove the function from the TS